Content.Server/Chemistry/Components/SolutionManager/RandomFillSolutionComponent.cs

```diff
@@ -0,0 +1,38 @@
+using Content.Shared.Chemistry.Components;
+
+namespace Content.Server.Chemistry.Components.SolutionManager;
+
+/// <summary>
+///     Fills a solution container randomly using a weighted list
+/// </summary>
+[RegisterComponent]
```

Comment:
Access

Reply:
`[RegisterComponent, Access(typeof(SolutionRandomFillSystem))]`
Like this?

Reply:
yeah